src/styles/header.scss

```diff
@@ -0,0 +1,66 @@
+@use './variables' as *;
+
+.moyo-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-sizing: border-box;
+  padding: 0 50px;
+  overflow: auto;
+
+  &__logotype {
+    width: $logo-header-size;
+    height: $logo-header-size;
+    margin-right: 40px;
+  }
+
+  &__list {
+    display: flex;
+    align-items: center;
+  }
+
+  &__item {
+    margin-right: 20px;
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    font-family: Roboto, sans-serif;
```

Comment:
The `font-family` is repeated here as well. If `Roboto, sans-serif` is the main font of the project, consider setting it globally.